templates/base_delete.html

```diff
@@ -20,7 +20,7 @@ <h1>
             {% block metadata_table %}{% endblock %}
             <form action="" method="post" class="form text-right">
                 {% csrf_token %}
-                <a href="{{ cancel_link }}" class="btn btn-link" type="submit">{% trans 'Cancel' %}</a>
+                <a href="{{ view.get_cancel_url }}" class="btn btn-link" type="submit">{% trans 'Cancel' %}</a>
```

Comment:
thinking about this a bit more, I think the cancel url for delete pages should take you back to the edit view you were just on, not the view page. do you agree @jeancochrane?

Reply:
That makes sense to me, but I'm not sure how it would affect this line of code. We would want to adjust that in the `get_cancel_url` methods implemented on children of `BaseDeleteView`, right?

Reply:
good catch. my latest commit implements an abstract class for delete views as well following the same pattern.